cosmo_tester/framework/test_hosts.py

```diff
@@ -436,6 +432,87 @@ def wait_for_manager(self):
                     )
 
 
+class _CloudifyDatabaseOnly(_CloudifyManager):
+    """
+    This class represents an instance of a Cloudify Database without a manager
+
+    Most of the inherited functions here are to avoid any incorrect usage of
+    the class since most of these functions rely on the manager existing on the
+    machine.
+    """
+
+    def __init__(self):
+        super(_CloudifyDatabaseOnly, self).__init__(upload_plugins=False)
+
+    def __str__(self):
+        return 'Cloudify Manager database only VM ({image}) [{index}:{ip}]' \
```

Comment:
The "manager" is part of the product's name, thus I felt it would be wise to reference it when addressing its database.
I might add a comma after "only" to for better clearance though